pkg/restore/restore.go

```diff
@@ -305,6 +317,7 @@ type restoreContext struct {
 	namespaceIncludesExcludes  *collections.IncludesExcludes
 	chosenGrpVersToRestore     map[string]ChosenGroupVersion
 	selector                   labels.Selector
+	selectors                  []labels.Selector
```

Comment:
Could you consider renaming the attribute `selectors` here and extract the logic to filter based on both `selector` and `selectors` into one func, and write some UT?

Reply:
Done, renamed to `orLabelSelectors`